gbfs-validator/nonSchemaValidation/translations.js

```diff
@@ -0,0 +1,103 @@
+const { getFileBody } = require('./utils')
+
+const TRANSLATED_FIELDS = {
+  system_information: [
+    ['name'],
+    ['short_name'],
+    ['operator'],
+    ['attribution_organization_name'],
+    ['terms_url'],
+    ['privacy_url']
+  ],
+  vehicle_types: [
+    ['vehicle_types', 'name'],
+    ['vehicle_types', 'make'],
+    ['vehicle_types', 'model'],
+    ['vehicle_types', 'description']
+  ],
+  station_information: [
+    ['stations', 'name'],
+    ['stations', 'short_name']
+  ],
+  system_regions: [['regions', 'name']],
+  system_pricing_plans: [
+    ['plans', 'name'],
+    ['plans', 'description']
+  ],
+  system_alerts: [
+    ['alerts', 'url'],
+    ['alerts', 'summary'],
+    ['alerts', 'description']
+  ],
+  geofencing_zones: [['geofencing_zones', 'features', 'properties', 'name']]
+}
+
+function checkTranslations(errors, data, languages, path, prefix = []) {
+  if (!data) {
+    return
+  }
+
+  if (path.length) {
+    if (Array.isArray(data)) {
+      for (const value of data) {
+        checkTranslations(errors, value, languages, path, prefix)
+      }
+      return
+    }
+
+    if (typeof data !== 'object') {
+      return
+    }
+
+    path = [...path]
+    let currentPath = path.shift()
+
+    checkTranslations(errors, data[currentPath], languages, path, [
+      ...prefix,
+      currentPath
+    ])
+    return
+  }
+
+  if (!Array.isArray(data)) {
+    return
+  }
+
+  for (const lang of languages) {
+    const filtered = data.filter((a) => a.language === lang)
```

Comment:
Array#some is more efficient here (and condition can be inlined)

Reply:
This is beyond my technical knowledge. Anyone, feel free to suggest an implementation. Thanks!